lib/developer_portal/lib/liquid/drops/message.rb

```diff
@@ -64,6 +64,10 @@ def recipients
           h(@model.recipients.first.try(:receiver).try(:org_name)) || ''
         end.html_safe
       end
+
+      def message
+        @message ||= @model.is_a?(MessageRecipient) ? @model.message : @model
```

Comment:
So here, do we ever get a `Message`?

Reply:
Yes, when we are on the Outbox (sent messages), the Drop wraps a `Message` object, and when we are on Inbox, then the Drop wraps a `MessageRecipient`.

Reply:
This is pure evil.